development-defaults.ini

```diff
@@ -54,23 +53,38 @@ black50 = "Black 50"
 black51 = "Black 51"
 black52 = "Black 52"
 black53 = "Black 53"
-tent01 = "Tent (Sandwhich) 01"
-tent02 = "Tent (Sandwhich) 02"
-tent03 = "Tent (Sandwhich) 03"
-tent04 = "Tent (Sandwhich) 04"
-tent05 = "Tent (Sandwhich) 05"
-tent06 = "Tent (Sandwhich) 06"
-tent07 = "Tent (Sandwhich) 07"
-tent08 = "Tent (Lakeside) 08"
-tent09 = "Tent (Lakeside) 09"
-tent10 = "Tent (Lakeside) 10"
-tent11 = "Tent (Lakeside) 11"
-orange01 = "Orange 01"
-orange02 = "Orange 02"
-orange03 = "Orange 03"
-orange04 = "Orange 04"
+black54 = "Black 00"
+primitive01 = "Tent 01 (Black)"
+primitive05 = "Tent 05 (Black) (Unmarked area behind pavilion)"
+primitive06 = "Tent 06 (Black)"
+primitive07 = "Tent 07 (?) (Black)"
+primitive08 = "Tent 08 (DJ Island)"
+primitive09 = "Tent 09 (DJ Island)"
+primitive10 = "Tent 10 (?) (DJ Island)"
+primitive11 = "Tent 11 (DJ Island)"
+primitive48 = “Tent 48 (Lakeside)
```

Comment:
Oh no! These are smartquotes instead of regular quotes, and they're unterminated. This part will still break things.